### DISCUSSION TITLE
Docs site theme switching

### DISCUSSION THREAD

**Discussion by croaky** [planetscale/discussion#12] [Bug Reports] [answered]
Hello, product is looking great! One small thing on the docs site:

* Switch theme
* Refresh page (or close tab, open docs site again)
* Content is gone
* Have to hard refresh to see content again

https://user-images.githubusercontent.com/198/118693187-af24e780-b7bf-11eb-95a0-0cc949684252.mov

**Comment by aquarapid** [comment, marked as answer]
Thanks @croaky!  Yes there has been some refresh issues with the docs site (not just related to theme switching);  we hope to get that straightened out soon.

**Reply by aquarapid** [reply]
We think we've sorted out most of the docs site refresh issues.  Can you confirm @croaky ?

**Reply by croaky** [reply]
Yes, it looks fixed to me now after clearing everything in cookies, IndexedDB, local storage etc., signing back in, switching themes, and refreshing. Thanks, @aquarapid!
